Tests/RxSwift/andTests.swift

```diff
@@ -0,0 +1,299 @@
+//
+//  andTests.swift
+//  RxSwiftExt
+//
+//  Created by Florent Pillet on 26/11/17
+//  Copyright © 2017 RxSwiftCommunity. All rights reserved.
+//
+
+import XCTest
+import RxSwift
+import RxTest
+
+fileprivate enum AndTestsError: Error {
+	case anyError
+}
+
+class andTests: XCTestCase {
+
+	fileprivate func runAndObserve(_ sequence: Maybe<Bool>) -> TestableObserver<Bool> {
+		let scheduler = TestScheduler(initialClock: 0)
+		let observer = scheduler.createObserver(Bool.self)
+		_ = sequence.asObservable().subscribe(observer)
+		scheduler.start()
+		return observer
+	}
+
+	func testSingle_OneTrueValue() {
+		let source = Observable.just(true)
+		let observer = runAndObserve(source.and())
+		let correct = [
+			next(0, true),
+			completed(0)
+		]
+		XCTAssertEqual(observer.events, correct)
+	}
+
+	func testSingle_OneFalseValue() {
+		let source = Observable.just(false)
+		let observer = runAndObserve(source.and())
+		let correct = [
+			next(0, false),
+			completed(0)
+		]
+		XCTAssertEqual(observer.events, correct)
+	}
+
+	func testSingle_EmptySequence() {
```

Comment:
Seems some tests follow a testSingle_ClassCase while some tests follow a testSingle_camelCase , I think the latter is more in tune with the rest of the tests as well as RxSwift's tests

Reply:
My Camels are erratic. Sometimes they are up, sometimes they are down. I'll give them a good spanking :)

Reply:
... and if they don't behave, will trade them for goats.